apps/events/deploy.yaml

```diff
@@ -168,7 +177,7 @@ spec:
             - name: GOOGLEWALLET_BASEURL
               value: "https://ch.tudelft.nl/events"
             - name: ADMINISTRATIONCOSTS
-              value: "0.35"
+              value: "0.00"
```

Comment:
Did you intend to remove the administration costs? It has been ping-ponging back and forth the last few years, but we should be consistent in our changes. I therefore suggest to only change this if it would be permanent.